Project.toml

```diff
@@ -7,8 +7,12 @@ version = "0.1.1"
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 KernelAbstractions = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
+LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
+NCDatasets = "85f8d34a-cbdd-5861-8df4-14fed0d494ab"
```

Comment:
Can we move these to the test env instead?

Reply:
Also, should we use ArtifactWrappers to avoid committing the nc files to the repo?